lib/index.js

```diff
@@ -67,9 +68,24 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
     optionallyAuthenticated,
     // FIXME add an implementation in bedrock-express
     boolParser(),
-    validate({query: 'account.getAccountsQuery'}),
+    intParser(),
+    validate({query: 'account.get'}),
     asyncHandler(async (req, res) => {
-      if(req.query.exists === true && req.query.email) {
+      /**
+       * @func getAccounts
+       * @param {Request} req
+       * @param {Response} res
+       * @param {Object} req.query
+       * @param {String} req.query.email - required to acces this route
+       * @param {Boolean} req.query.exists - used by Registration
+       * @param {Number} req.query.limit
+       * @param  {String} req.query.cursor - an account DID
+       * @description multi-purpose function that can used
+       * by visitors not logged in to confirm an email exists
+       * in the system or by admins to access sensitive account
+       * information
+       */
+      if(req.query.exists === true) {
```

Comment:
We still need to check within `exists` for the presence of an `email` filter ... unless you change the validator to catch that. Otherwise `email` could be undefined in what gets passed below.

Reply:
validator requires email.

Reply:
yeah I double checked email is required for this route.

Reply:
It needs to only be required when `exists = true` or else you won't be able to paginate accounts that have different emails.

Reply:
@aljones15 here's some psuedocode for the validator based on what I gather you *might* need here.
```
{
  anyOf: [{
    // some schema with required properties
  }, {
    // another schma with different set of required properties
  }
{
```
there's also `oneOf`, but that forces the validator to check ALL schemas to ensure that the document matches *exactly* one of the schemas.  Usually, you just want to pass if any of the schemas are valid.